.travis.yml

```diff
@@ -30,9 +30,14 @@ install:
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
   - conda install -y -c conda-forge cmake
+  - pip install cpp-coveralls
```

Comment:
Using conda and pip to manage packages seems odd. Can we just use one or the other?

Reply:
Yes and no. I looked at this.  cpp-coveralls is available via conda, but not from a major provider. `pip` seemed to be a safer path because we know that pypi will have the dep.